packages/gel/test/transaction.test.ts

```diff
@@ -42,16 +42,31 @@ async function run(test: (con: Client) => Promise<void>): Promise<void> {
 beforeAll(async () => {
   await run(async (con) => {
     await con.execute(`
+      CREATE MODULE test;
+
       CREATE TYPE ${typename} {
         CREATE REQUIRED PROPERTY name -> std::str;
       };
+
+      CREATE TYPE ${typename}Conflict {
+          CREATE REQUIRED PROPERTY tmp -> std::str {
+              CREATE CONSTRAINT exclusive;
+          }
+      };
+
+      CREATE TYPE ${typename}ConflictChild extending ${typename}Conflict;
```

Comment:
Preemptively copying the test setup from the python work on `PreferRepeatableRead` knowing that we'll be adding this stuff later.